### DISCUSSION TITLE
[recordrtc]: getSeekableBlob should be static

### DISCUSSION THREAD

**Discussion by SirUppyPancakes** [DefinitelyTyped/DefinitelyTyped#55662] [Issues with a @types package] [answered]
As seen in the docs (on the right) here:
https://recordrtc.org/index.html

`getSeekableBlob` is a global and should be typed as `static` like other globals (a common example being `invokeSaveAsDialog`)

`getSeekableBlob`:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/8173ef4bfd4968c60bdca5a491f91f18d92c5ada/types/recordrtc/index.d.ts#L269

`invokeSaveAsDialog`:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/8173ef4bfd4968c60bdca5a491f91f18d92c5ada/types/recordrtc/index.d.ts#L312

[a comment by an automated account is not shown]

**Comment by k-yle** [comment, marked as answer]
Hey @SirUppyPancakes, I've fixed this in #55666

**Reply by SirUppyPancakes** [reply]
Awesome, thanks for the quick turnaround!
